lib/biran/config_defaults.rb

```diff
@@ -47,23 +47,35 @@ def config_dir
 
     def local_config_file
       ENV['BIRAN_LOCAL_CONFIG_FILE'] ||
```

Comment:
Was sure if I should change this, I left it for now... This would be the whole path. As documented, this variable should really be the one listed below in the filename method. I doubt this is being used, but leaving here as it doesn't hurt anything.